src/couch_replicator_scheduler.erl

```diff
@@ -50,6 +50,7 @@
 -record(job, {
           id :: job_id(),
           rep :: #rep{},
+          continuous :: boolean(),
```

Comment:
this information is inside the `rep` field, I don't think we should make a copy.